harness/testAtomics.js

```diff
@@ -44,6 +39,11 @@ function testWithAtomicsInBoundsIndices(f) {
     var good_indices = [
         (view) => 0/-1, // -0
         (view) => '-0',
+        (view) => undefined, // 0
+        (view) => Number.NaN, // 0
+        (view) => 3.5,
+        (view) => '3.5',
+        (view) => ({ password: "qumquat" }),
         (view) => view.length - 1,
         (view) => ({ valueOf: () => 0 }),
         (view) => ({ toString: () => '0', valueOf: false }) // non-callable valueOf triggers invocation of toString
```

Comment:
you can include `false` to improve coverage for the ToIndex operation here.

`-0.9` should work as a good index as well, weird, but it works.